js/src/Collection/TodoCollection.js

```diff
@@ -0,0 +1,18 @@
+var Backbone = require('backbone');
+
+var TodoModel = require('../Model/TodoModel.js');
+
+var TodoCollection = Backbone.Collection.extend({
+    initialize: function(models, options){
```

Comment:
Could you add a comment to this constructor? Comment format is usually:
```
/**
 * Comment about overall function.
 *
 * @param {data type} param1 Comment about param1
 * 
 * @return {data type} Comment about return
 */
```
So for example:
```
/**
 * Constructor. Initializes a new collection for todos that have the specified todo list ID.
 * 
 * @param {Array.<TodoModel>} models Array of models.
 * @param {Object} options The options with keys:
 *     @param {string} options.id ID for this todo list (todo collection).
 */
```